stable/s3-proxy/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: "3.1.0"
+appVersion: "3.2.0"
 description: A Helm chart for Kubernetes to deploy S3-Proxy
 name: s3-proxy
-version: 2.1.0
+version: 2.3.1
```

Comment:
Can you change the version to 2.2.0 please ?

Reply:
done :+1: